docs/getting_started/logging-in.md

```diff
@@ -70,6 +70,7 @@ RMACC members can obtain a CURC account by completing the following steps
     - your department
     - your first and last name
     - your preferred email address for communication
+    - your NSF discipline (refer to the list of disciplines [here](https://osp.unm.edu/pi-resources/nsf-research-classifications.html))
```

Comment:
I am curious what people think. I am kinda against saying check out the link `here`. I would prefer just to link an appropriate title. For this one, you could just make `list of disciplines` the link. Additionally, the link you provide goes to a table that is not owned by NSF. People might also give you the NSF line number instead of the discipline. 

An idea that came up while talking to Mohal is that maybe we should create our own table that is available via a drop-down. I like this idea better because we have full control over it.

Reply:
Resolved the first part of the comment: moved the link to 'list of disciplines' instead of 'here'

Reply:
Thank you!